specs/signature-specification.md

```diff
@@ -308,5 +360,7 @@ Alternatively, an implementation of Notary v2 can choose not to implement plugin
 [annotation-rules]: https://github.com/opencontainers/image-spec/blob/main/annotations.md#rules
 [oci-descriptor]: https://github.com/opencontainers/image-spec/blob/main/descriptor.md
 [ietf-rfc3161]: https://datatracker.ietf.org/doc/html/rfc3161#section-2.4.2
```

Comment:
Somehow, I cannot see these links in the webpage. I think these links are not successfully rendered in the markdown, hence:
```suggestion
annotation-rules: https://github.com/opencontainers/image-spec/blob/main/annotations.md#rules</br>
oci-descriptor: https://github.com/opencontainers/image-spec/blob/main/descriptor.md</br>
ietf-rfc3161: https://datatracker.ietf.org/doc/html/rfc3161#section-2.4.2</br>
```

Reply:
This is the correct usage for replacing the reference with the links for a markdown file. So I will keep it no changed.